spec/services/decisions/case_decision_tree_spec.rb

```diff
@@ -7,27 +7,24 @@
     instance_double(
       CrimeApplication,
       id: '10',
-      applicant: applicant_double,
+      applicant: instance_double(Applicant),
       case: kase,
       kase: kase,
```

Comment:
Wondering if these need a comment? Looks really odd.

I only half believe I understand myself. case is on the form_object and kase is on the crime_application?

Reply:
Thank you!